Bottom: 9fff06c4a1e7a460476f5b452278dc636cd49064
Top:    04aa2e93e42d3bdee3b0c6f971bf16436edef6a9
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-31 09:32:24 -0400

Refresh of base

---

diff --git a/arch/arm/mach-lpc31xx/i2c.c b/arch/arm/mach-lpc31xx/i2c.c
index 1515c3e..ea1e4b2 100644
--- a/arch/arm/mach-lpc31xx/i2c.c
+++ b/arch/arm/mach-lpc31xx/i2c.c
@@ -80,6 +80,8 @@ static u32 calculate_input_freq(struct platform_device *pdev)
 	return (FFAST_CLOCK/1000000);
 }
 
+/* disable this until device tree patch */
+#if 0
 
 static struct i2c_pnx_data lpc_pnx_data0 = {
 	.name = I2C_CHIP_NAME "0",
@@ -154,3 +156,8 @@ void __init lpc313x_register_i2c_devices(void)
 	platform_add_devices(devices, ARRAY_SIZE(devices));
 }
 
+#else
+void __init lpc313x_register_i2c_devices(void)
+{
+}
+#endif
